### PR TITLE
Added German translation

### PR DESCRIPTION
**Pull request by mrbesen**

### CODE REVIEW

File: locale/de/locale.cfg [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[timecontrol]
+speed-changed=Geschwindigkeit beträgt nun __1__x.
+speed-changed-by=__2__ Hat die Geschwindigkeit auf __1__x gesetzt.
+speed-reset=Die Geschwindigkeit ist wieder normal.
+speed-reset-by=Geschwindigkeit wurde von __2__ zurückgesetzt.
+
+gui-speed-warping=Time Warp: __1__x
+gui-speed-normal=Time Warp: disabled
```
Comment (SuperSandro2000) [inline locale/de/locale.cfg:8]: I would translate disabled too. How do you like ausgeschalten? Or aus?

File: locale/de/locale.cfg [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[timecontrol]
+speed-changed=Geschwindigkeit beträgt nun __1__x.
+speed-changed-by=__2__ Hat die Geschwindigkeit auf __1__x gesetzt.
```
Comment (SuperSandro2000) [inline locale/de/locale.cfg:3]: Should hat be lowercased?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
I would like to see some more changes.

**Comment by dewiniaid** [comment]
Closing in favor of #2

**Pull request closed without merging**